x-pack/plugins/alerting/server/usage/lib/get_telemetry_from_alerts.test.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { elasticsearchServiceMock, loggingSystemMock } from '@kbn/core/server/mocks';
+import { getTotalAlertsCountAggregations } from './get_telemetry_from_alerts';
+
+const elasticsearch = elasticsearchServiceMock.createStart();
+const esClient = elasticsearch.client.asInternalUser;
+const logger: ReturnType<typeof loggingSystemMock.createLogger> = loggingSystemMock.createLogger();
+
+describe('kibana index telemetry', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should return total alert couts and alert counts by rule type id', async () => {
```

Comment:
Feels like we should have a test for no results at all, separate from the "error" test below this

Reply:
Done